From 6325f548bf952e2e48f867ad0de2b3b1ca54e52f Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:10:50 +0800
Subject: [PATCH 1498/1594] MLK-11123-25 ARM: dts: disable otg features for
 imx6sx-17x17-arm2 board

commit 7d37abe93728d8b5a4ce9df35b562c820e7b1557 from
git://git.freescale.com/imx/linux-2.6-imx.git

Disable all otg features for otg1 port of imx6sx-17x17-arm2 board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
(cherry picked from commit 46d2433c7e8e8b89352c823a6ccf267d3ac7882b)
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 936c7a6..948c93a 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -497,6 +497,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usbotg1_1>;
 	disable-over-current;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.7.5.4

